src/object/powerup.cpp

```diff
@@ -96,7 +93,7 @@ PowerUp::get_default_sprite_name() const
     case COFFEE:
       return "images/powerups/retro/coffee.png";
     case HERRING:
-      return "images/powerups/retro/golden_herring.png";
+      return "images/powerups/retro/golden_herring.sprite";
```

Comment:
Why was the herring changed to a sprite but not the coffee? Does the herring glow?

Reply:
Yes, the herring glows.